### PR TITLE
Add missing CQB CBM fighting styles

### PR DESCRIPTION
**Pull request by aMegaSloth**
#### Summary
Bugfixes "Include newer fighting styles to the CQB CBM"

#### Purpose of change
Updates the close quarter battle CBM further by adding new fighting styles introduced, but were probably overlooked into adding to the CBM. 

#### Describe the solution
Update the Bionics CBM with the new fight styles.

#### Testing
Load up a new safe, install the combat cbm. Verify new fighting styles are listed

### CODE REVIEW

File: data/json/bionics.json [hunk 1]
```diff
@@ -678,17 +678,22 @@
...
       "style_dragon",
       "style_eskrima",
       "style_fencing",
+      "style_medievalpole",
       "style_judo",
       "style_karate",
       "style_krav_maga",
       "style_leopard",
+      "style_lizard",
+      "style_scorpion",
```
Comment (TheShadowFerret) [inline data/json/bionics.json:695]:
```suggestion
```
These two were obsoleted from vanilla and moved to Mythical Martial Arts

Comment (aMegaSloth) [reply data/json/bionics.json:695]: Both the style lizard and scorpion are listed in martialarts_fictional. Is there any reason why it wasn't a complete migration?

Comment (TheShadowFerret) [reply data/json/bionics.json:695]: Straight removal of them from vanilla files would break saves from people who had those arts pre-migration.

Comment (aMegaSloth) [reply data/json/bionics.json:695]: Hmmm.

Digging though some of the files at all it doesn't seem like those fight styles are trying to be retired at all. They just aren't 'real' fighting styles  used in the real world. the books to learn them are still in the drop tables, and if they were being migrated, I would imagine they would at least be pulled into an obsolete json file.

Anyone else have an opinion on whether fictional fight styles should be included?

Comment (Fris0uman) [reply data/json/bionics.json:695]: You're duplicating the scorpion entry, see line 701. Also the martial art book for those style are not in any item groups, so they don't spawn and are as far as I can tell on their way out.

**Comment by Fris0uman** [comment]
Closing as stale, feel free to reopen if you plan to finish it

**Pull request closed without merging**